test/jdk/sun/security/x509/X509CertImpl/CertExtensions.java

```diff
@@ -27,8 +27,6 @@
  * @summary validating getExtensionValue and getKeyUsage with specified and
  *          unspecified extensions on the X509Certificate.
  * @library /test/lib
- * @modules java.base/sun.security.x509
- *          java.base/sun.security.util
```

Comment:
Why remove these `@modules` lines? They are still necessary to access internal classes.